### DISCUSSION TITLE
Directory of example Beaker web apps?

### DISCUSSION THREAD

**Discussion by omgitsraven** [beakerbrowser/beaker#1863] [Q&A]
Is there anywhere to find examples of existing Beaker sites or web apps, to demonstrate the APIs being used in real-world contexts?

**Comment by pfrazee** [comment]
I hope you don't mind, I moved this issue to discussions. Also, thanks for the good set of issues you opened, I'll tackle them next week.

We have a "leftover" list at https://userlist.beakerbrowser.com from when we were still thinking about social ideas. That's not a bad place to dig around.

If anybody creates their own hub, please let me know.
